packages/calcite-components/src/components/popover/popover.tsx

```diff
@@ -145,6 +146,12 @@ export class Popover
   /** Specifies the heading level of the component's `heading` for proper document structure, without affecting visual styling. */
   @property({ type: Number, reflect: true }) headingLevel: HeadingLevel;
 
+  /**
+   * Specifies whether focus should move to the popover when the focus trap is activated.
+   *  `@internal`
+   */
+  @property({ type: Boolean }) initialFocusTrapFocus: FocusTargetOrFalse;
```

Comment:
WDYT about `focusTrapInitialFocus`? I'll leave it up to you.

Reply:
woops, didn't see this comment.  Ah well